meteor/lib/api/prompter.ts

```diff
@@ -75,14 +75,14 @@ export namespace PrompterAPI {
 				parts: [],
 			}
 
-			_.each(parts, (part) => {
+			parts.forEach((part) => {
 				const partData: PrompterDataPart = {
 					id: part._id,
 					title: part.title,
 					pieces: [],
 				}
 
-				_.each(part.getAllPieces(), (piece) => {
+				part.getAllPieces().forEach((piece) => {
```

Comment:
changing this inner loop to a for-loop might have slight improvements in performance.

Reply:
Switched to a `for ... of` loop.